<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {background-color: rgb(255, 255, 255);}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="simulationImageLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1590</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(173, 216, 230)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="questionLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>650</y>
      <width>1291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>put the questions here!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fireButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>570</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fire</string>
    </property>
   </widget>
   <widget class="QToolBox" name="slidersBox">
    <property name="geometry">
     <rect>
      <x>1077</x>
      <y>20</y>
      <width>171</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {color: green;}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="gravityPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>92</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Gravity</string>
     </attribute>
     <widget class="QSlider" name="gravitySlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="gravitySpinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="initialVelocityPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>88</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Initial Velocity</string>
     </attribute>
     <widget class="QSlider" name="initialVelocitySlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="value">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="initialVelocitySpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSpinBox {color: green;}</string>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="heightPage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>88</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Height</string>
     </attribute>
     <widget class="QSlider" name="heightSlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="maximum">
       <number>150</number>
      </property>
      <property name="value">
       <number>3</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="heightSpinBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="anglePage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>88</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Angle:</string>
     </attribute>
     <widget class="QSlider" name="angleSlider">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>160</width>
        <height>16</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-90</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="angleSpinBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-90</number>
      </property>
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QRadioButton" name="choice1Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>611</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>put answer 1 here</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choice2Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>720</y>
      <width>671</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>put answer 2 here</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choice3Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>750</y>
      <width>611</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>put answer 3 here </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choice4Button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>780</y>
      <width>611</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>put answer 4 here</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>680</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QLabel" name="questionNumberLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Question:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>570</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="cannonLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="groundLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>500</y>
      <width>1590</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,128,0)
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cannonBallLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color: rgba(173, 216, 230, 0%)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="submitButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>690</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>submit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="formulaButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>690</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Formula Page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetQuizButton">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>690</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset Quiz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>570</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Main Menu</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
